ckanext/xloader/tests/samples/sample_with_mixed_quotes.csv

```diff
@@ -0,0 +1,136 @@
+Category,Category name,Priority,Initiative name,Investment objectives,Primary digital priority,Initiative stage,Actual start date,Approved end date,Date data current at,Percentage complete,Overall status,Project commencement allocation,Approved expenditure,Actual cost to date,Scope change event,Cost re-evaluation event,Delivery delay event,Project journey and reasons for variance,Learn more (URL)
```

Comment:
could you redact information with lipsum data so that we are not polluting a plugin with data that is not pertaining to it.

Reply:
Replaced actual department and program names with Monty Python.